week-02/assessment/gomoku/src/test/java/learn/gomoku/game/GameControllerTest.java

```diff
@@ -0,0 +1,28 @@
+package learn.gomoku.game;
+
+import learn.gomoku.players.RandomPlayer;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameControllerTest {
```

Comment:
Go ahead and remove this test class now that the method it was testing has been removed.